### ISSUE TITLE
Some HUD textures missing

### ISSUE THREAD

**Issue by TheNickOfMax** [MorsGames/sm64plus#85]
I've built the game as recommended in the [Manual Building guide](https://github.com/MorsGames/sm64plus/wiki/Manual-Building-Guide) but for some reason the "Peach" word in the start letter and some HUD elements (aka camera symbol, mario's head camera symbol, coins symbol etc.)

**Comment by Kaiju2019** [comment]
Yeah. It happened to me too. The little red rug thingy at the door is a texture not found symbol too.

**Comment by BrunoTheMad** [comment]
Experiencing the same issue here, most of the hud elements are showing up as missing textures.

**Comment by themirman** [comment]
same issue, I tried what it said to do in the FAQ (send to GFX and delete GFX folder) neither worked and deleting the folder made it worse. Anyone find a fix?

**Comment by Kaiju2019** [comment]
what in the world

**Comment by Reual4412** [comment]
Alright, I need everyone to check this please.
Keep the build folder in the exact same directory with the bug. Rename us_pc.
For whatever reason, that fixes it. and renaming it back for me at least, to us_pc manually, causes the issue to reappear.

**Comment by jenrik** [comment]
this issue can be solved by passing the path to the gfx folder in the build directory as the first argument.

Example: `./build/us_pc/sm64.us ./build/us_pc/gfx`

Source: [the FAQ](https://github.com/MorsGames/sm64plus/wiki/Frequently-Asked-Questions#q-some-of-the-textures-appear-black-and-purple-how-do-i-fix-this)

**Comment by TripleLLL9** [comment]
> this issue can be solved by passing the path to the gfx folder in the build directory as the first argument.
> 
> Example: `./build/us_pc/sm64.us ./build/us_pc/gfx`
> 
> Source: [the FAQ](https://github.com/MorsGames/sm64plus/wiki/Frequently-Asked-Questions#q-some-of-the-textures-appear-black-and-purple-how-do-i-fix-this)

for some reason that just made it worse for me :P

**Comment by fortpolio** [comment]
The problem is not fixed by that merge.

**Comment by MorsGames** [comment]
The big issue I have is that I was not able to replicate the issue at all. I don't have a Steam Deck, so I only tried on my main PC... and yeah, it just works fine on my end. So I'm completely stumped unfortunately.

**Comment by Nitr4m12** [comment]
It really seems to happen at random. Sometimes it gets fixed by closing and reopening the game over and over.

**Comment by LSeelig** [comment]
> The big issue I have is that I was not able to replicate the issue at all. I don't have a Steam Deck, so I only tried on my main PC... and yeah, it just works fine on my end. So I'm completely stumped unfortunately.

I can confirm this issue on my PC (Fedora 42, Intel i5-1240p). Since steam deck is affected, I doubt amd/intel matters. Wonder if the problem is in the build or running it.
Is it possible to compile a Linux build on Windows?
